package.json

```diff
@@ -12,7 +12,7 @@
     "jquery": "^3.1.1",
     "jquery-ui": "^1.12.1",
     "lodash": "^4.16.4",
-    "metabolica": "github:DD-DeCaF/metabolica-ui#devel",
+    "metabolica": "github:biosustain/metabolica-ui",
```

Comment:
I had to revert, because I wasn't able to use it in `decaf-metabolica`like that.